Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -509,6 +516,10 @@ function xASL_wrp_Quantify(x, PWI_Path, pathOutputCBF, M0Path, SliceGradientPath
     xASL_spm_deformations(x, {x.P.Path_CBF}, {x.P.Pop_Path_CBF}, [], [], AffineTransfPath, x.P.Path_y_ASL);
 	if xASL_exist(x.P.Path_TT,'file')
 		xASL_spm_deformations(x, {x.P.Path_TT}, {x.P.Pop_Path_TT}, [], [], AffineTransfPath, x.P.Path_y_ASL);
+    end
+    
+    if xASL_exist(x.P.Path_TExch,'file')
```

Comment:
Do we have this for other outputs, e.g. BASIL ATT?

Reply:
I don't understand. Yes - we have this for TExch and ATT (see the line above).

Reply:
Using ATT instead of TT for multi-PLD: 6eeff88f